superset-frontend/src/explore/controlPanels/DeckGeojson.js

```diff
@@ -43,23 +56,23 @@ export default {
           },
           null,
         ],
-        ['row_limit', 'filter_nulls'],
+        ['row_limit', filterNulls],
         ['adhoc_filters'],
       ],
     },
     {
       label: t('Map'),
       controlSetRows: [
-        ['mapbox_style', 'viewport'],
-        // TODO ['autozoom', null],
+        ['mapbox_style', viewport],
+        // TODO [autozoom, null], // import { autozoom } from './Shared_DeckGL'
```

Comment:
complete?

Reply:
I don't know... the TODO is something you added a year ago. The control exists, but I didn't add it in there to see if it works. Added functionality seems out of scope, so I just noted where the control could be imported from.

Reply:
lol. ok. then just leave it like that.